package.json

```diff
@@ -26,7 +26,7 @@
   "author": "Ben Zörb <benjamin.zoerb@jvm.de>",
   "license": "MIT",
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
```

Comment:
We need to increment the version with all those changes.
Is it done later when the whole thing is released?
This is a breaking change, is it not?
So we would require a new mayor version, right?

Reply:
Yes, would be minor but as we increased the required node version this needs to be major. Version will be bumped in the release process